kernel/core/io/file.hpp

```diff
@@ -24,6 +24,8 @@
 #include <cmath>
 #include <vector>
 #include <utility>
+#include <set>

```

Comment:
in hpp importing set and algorithm for find_if. Looks to be fastest way to do either of its kind

Reply:
There's probably no need to have it in the hpp file, as it is only referenced in file.cpp. Move it to this file please (keep foreign imports* as clean as possible)

* Foreign imports: Imports that you receive via an imported file.